### ISSUE TITLE
Compatibility with Python 3.9 - cgi.escape

### ISSUE THREAD

**Issue by johanvdw** [FOSDEM/volunteers#43]
```
g gunicorn[1243]:   File "/var/opt/app/fosdem_volunteers/tool/volunteers/views.py", line 33, in <module>
g gunicorn[1243]:     from cgi import escape
g gunicorn[1243]: ImportError: cannot import name 'escape' from 'cgi' (/usr/lib/python3.9/cgi.py)

```

**Comment by johanvdw** [comment]
we can probably replace it by `from django.utils.html import escape`

**Comment by johanvdw** [comment]
Closed in e9822bc8800
